torch/_inductor/config.py

```diff
@@ -1634,6 +1634,9 @@ class test_configs:
 
     graphsafe_rng_func_ignores_fallback_random = False
 
+    # TODO - temporary config before enabled by default
+    global_tiling_analysis: bool = False
```

Comment:
Have we done a CI/benchmark run with this on to check for issues?

Reply:
I'm still working through a few bugs etc..

Reply:
I get a 3% improvement in timm training, and 9% torchbench speedup. although not clear how much is noise. I did collect some different kernels and perf differences while looking, where the new logic was always doing better. https://docs.google.com/document/d/12_cep-9_VLhnW-V0mswAmbU_QuTVoiwsjyWTvsA91nE/edit?tab=t.0